src/avfoundation.cs

```diff
@@ -3854,7 +3999,7 @@ interface AVAssetWriterInput {
 #if XAMCORE_2_0
 		[Internal]
 #endif
-		[Advice ("Use constructor or 'Create' method.")]
+		[Advice ("Use constructor or Create method")]
```

Comment:
Did we do this un purpose? I though we wanted to use quotes.

Reply:
no, merge likely thought the one from `xcode9.1` was the most recent